scripts/install.ps1

```
@@ -0,0 +1,617 @@
+    if (-not [string]::IsNullOrWhiteSpace($HOME)) {
+        return $HOME
+    }
+    $profileHome = [Environment]::GetFolderPath("UserProfile")
+    if (-not [string]::IsNullOrWhiteSpace($profileHome)) {
+        return $profileHome
+    }
+    Fail "Unable to resolve user home directory."
+}
+function Resolve-InstallDir([string]$Value) {
+    if ([string]::IsNullOrWhiteSpace($Value)) {
+        $home = Get-UserHome
```

PSScriptAnalyzer (1.25.0)

[error] The Variable 'home' cannot be assigned since it is a readonly automatic variable that is built into PowerShell, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)